src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { FC } from 'react';
+import { ItemsList } from '../ItemsList/ItemsList';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: React.Dispatch<React.SetStateAction<number>>,
+};
+const itemList = (total: number, perPage: number, currentPage: number) => {
+  // eslint-disable-next-line max-len
+  return new Array(total).fill(0).map((_, i) => i + 1).splice(currentPage * perPage - perPage, perPage);
```

Comment:
```suggestion
  return new Array(total)
    .fill(0)
    .map((_, i) => i + 1)
    .splice(currentPage * perPage - perPage, perPage);
```
you can just format it smth like this